repositories.yaml

```diff
@@ -226,14 +242,17 @@ applications:
         app_id: org.mozilla.firefox
         app_channel: release
         additional_dependencies:
+          # TODO: This is using beta and not release because...
```

Comment:
Should we figure this out before landing?

Reply:
I admit to hoping you'd know what's going on here, but I think @badboy found the answer in the code that this actually _should_ be the `-release` variant.